### ISSUE TITLE
Stops Downloading for no reason

### ISSUE THREAD

**Issue by 10lalib** [fmd-project-team/FMD#956]
When FMD is downloading new chapter for some unknown reason it stops downloading, while my internet is still working fine.
I have to keep stopping the download and resuming all the time for it to continue downloading.
Any Help??
ps: the network meter on the top right also shows 0 bps.
Thanks

**Comment by chrysalist** [comment]
Yeah it happens when the chapter check is going on too. I usually just select the chapters again and download

**Comment by 10lalib** [comment]
Thank you for your reply. I will take note for the future but currently in my case whether it is checking for new chapters or not i am getting this problem :(
